db/migrate/20170106122400_link_documents_and_editions.rb

```diff
@@ -0,0 +1,12 @@
+class LinkDocumentsAndEditions < ActiveRecord::Migration[5.0]
+  disable_ddl_transaction!
+
+  def change
+    add_column :content_items, :document_id, :integer
+    add_foreign_key :content_items, :documents
```

Comment:
These queries should be within a transaction, so I think a separate migration for them would be better.

I believe we also don't get an index implicitly by adding a foreign key so should add an index here too